ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -417,6 +417,44 @@ default_interfaces:
         inter_domain: < true | false | Default -> True >
 ```
 
+### BGP & IP-VPN Gateway
+
+```yaml
+< node_type_key >:
+
+  defaults:
+    # Node is acting as IP-VPN Gateway for EVPN to MPLS-IP-VPN Interworking | Optional.
+    # The BGP peer group used for this is "bgp_peer_groups.mpls_overlay_peers".
+    # L3 Reachability is required for this to work, the preferred method to establish underlay connectivity is to use core_interfaces.
+    ipvpn_gateway:
```

Comment:
Do we need some checks and also description here that it only works for evpn-vxlan fabrics? Or will it work for evpn-mpls -> ipvpn ?

Reply:
Have to investigate if this is only for vxlan to mpls or if will work for evpn-mpls to ipvpn as well...

Reply:
Will work for evpn-mpls to ipvpn. Have made updates discussed offline.